tests/Issues/AutoImport/Fixture/DocBlock/two_routes_with_comment_before.php.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Rector\Core\Tests\Issues\AutoImport\Fixture\DocBlock;
+
+class TwoRoutesWithCommentBefore
+{
+    /**
+     * Testsssssssssss
+     *
+     * @\Symfony\Component\Routing\Annotation\Route("/first", methods={"GET"})
+     * @\Symfony\Component\Routing\Annotation\Route("/second", methods={"GET"})
+     */
+    public function some(): Response
+    {
+        return new Response();
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Core\Tests\Issues\AutoImport\Fixture\DocBlock;
+
+use Symfony\Component\Routing\Annotation\Route;
+class TwoRoutesWithCommentBefore
+{
+    /**
+     * Testsssssssssss
```

Comment:
this space between comment and `@Route` removed after change is expected as trimmed on redefined `PhpDocTextNode`